app/application.py

```diff
@@ -52,6 +51,24 @@ def hello_world():
     return render_template('_temp.html', _theme='default')
 
 
+def build_survey_data(statusFilter):
+
+    # TODO - Derive the Party Id
+    party_id = "3b136c4b-7a14-4904-9e01-13364dd7b972"
+
+    # TODO - Add security headers
+    # headers = {'authorization': jwttoken}
+    headers = {}
+
+    # Call the API Gateway Service to get the To Do survey list
+    url = Config.API_GATEWAY_SURVEYS_URL + 'todo/' + party_id
+    req = requests.get(url, headers=headers, params=statusFilter, verify=False)
```

Comment:
You are making a call to requests.get which depends on an external application. So we should wrap this up in an except clause. It sh*t can happen it will! So we can protect against that by catching the exception and presenting a page with maybe "Currently have issues getting your cases please contact Bill Gates!" or something like that.

Check out the method 'fetch.token' which does a request.get under the hood. We catch all exceptions and display some sense to the user. It's not perfect, but better than nothing! :-)